plugins/pastefromword/plugin.js

```diff
@@ -66,9 +66,11 @@
 					// dataValue should be used.
 					mswordHtml = dataTransferHtml || data.dataValue,
 					pfwEvtData = { dataValue: mswordHtml },
-					wordRegexp = /(class=\"?Mso|style=(?:\"|\')[^\"]*?\bmso\-|w:WordDocument|<o:\w+>|<\/font>)/;
+					officeMetaRegexp = /<meta\s*name=("|')?generator("|')?\s*content=("|')?microsoft/gi,
```

Comment:
Let's use unnamed groups, just like one line below (so that it is consistenet) e.g. `<meta\s*name=(?:"|')?generator(?:"|')?\s*content=(?:"|')?microsoft`